runner/runner/main.py

```diff
@@ -41,7 +41,7 @@
 # Environment variables for configuring the GPU
 env_gpu : Mapping[str, str] = dict(
     __GL_MaxFramesAllowed="1", # Double buffer framebuffer
-    __GL_SYNC_TO_VBLANK="0",   # Block on vsync
```

Comment:
Why we are changing this?

Reply:
Setting this variable to 1 essentially forces `glXSwapBuffers` to synchronize with the display device's vertical blanking period (vsync). Otherwise, swap could happen without waiting for vsync. The former behavior makes more sense by default, right? [reference](https://download.nvidia.com/XFree86/Linux-x86_64/352.79/README/openglenvvariables.html)

Reply:
agree